hal/stm32f4/bootstrap.c

```diff
@@ -117,6 +117,9 @@ void boot(const bootstrap_prog_header *header)
 	// set cpu to run vector table from ram
 	NVIC_SetVectorTable(NVIC_VectTab_RAM, 0);
 
+	// re-initialise the stack pointer
+    __set_MSP((uint32_t)isr_vector_ram[0]);
+
```

Comment:
oh can you sort out the tab/space here ... it looks like it is using spaces or something ?